tests/fsharpqa/Source/Conformance/DeclarationElements/CustomAttributes/Basic/W_StructLayoutSequentialPos_ClassExpliCtr.fs

```diff
@@ -1,15 +1,14 @@
-// #Regression #Conformance #DeclarationElements #Attributes 
+// #Regression #Conformance #DeclarationElements #Attributes
 // Regression: FSB 4014
 // Need to tighten up our imlementation of StructLayout.Sequential.
-//<Expects id="FS0009" span="(9,10-9,12)" status="warning">Uses of this construct may result in the generation of unverifiable \.NET IL code\. This warning can be disabled using '--nowarn:9' or '#nowarn "9"'\.</Expects>
 
-module PositiveTests = 
+module PositiveTests =
 
     [<System.Runtime.InteropServices.StructLayout(System.Runtime.InteropServices.LayoutKind.Sequential)>]
```

Comment:
In the corresponding env.lst file you can set compiler options. Please set --warningsaserrors (or how is it called?)